### PR TITLE
Add test: ASTCreateMaskingPolicyQuery::formatImpl has zero test covera

### PR DESCRIPTION
**Pull request by clickgapai**
_Found via ClickGap automated review. Please close or comment if this is incorrect or needs adjustment._

Adds test coverage for 1 untested code path(s), found during automated review of [PR #97594](https://github.com/ClickHouse/ClickHouse/pull/97594).

**1. ASTCreateMaskingPolicyQuery::formatImpl has zero test coverage for output verification**
  `src/Parsers/Access/ASTCreateMaskingPolicyQuery.cpp:82`, `src/Parsers/Access/ASTCreateMaskingPolicyQuery.cpp:84`

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### Changelog entry (a [user-readable short description](https://github.com/ClickHouse/ClickHouse/blob/master/docs/changelog_entry_guidelines.md) of the changes that goes into CHANGELOG.md):
Not applicable — test-only change.

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

---
_Found during review of [PR #97594](https://github.com/ClickHouse/ClickHouse/pull/97594)._

### CODE REVIEW

File: tests/queries/0_stateless/04070_masking_policy_format_coverage.sql [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
+-- Coverage for ASTCreateMaskingPolicyQuery::formatImpl (lines 66-108)
+-- Exercises: name, ON db.table, IN storage_name, WHERE, PRIORITY, ALTER paths
+
+-- Basic CREATE with database-qualified table
```
Comment (bharatnc) [inline tests/queries/0_stateless/04070_masking_policy_format_coverage.sql:5]: Support only exists for clickhouse cloud only, so doesn't make sense to add more tests here.
